src/Command/SymlinksCommand.php

```diff
@@ -206,11 +212,11 @@ private function symlink($source, $target, $rootDir, OutputInterface $output)
      */
     private function validateSymlink($source, $target, $rootDir)
     {
-        if ($source == '') {
+        if ($source === '') {
```

Comment:
Should be `('' === $source)`, although of course this is not related to this PR :)